src/components/app/details/appDetails/utils.tsx

```diff
@@ -664,4 +728,9 @@ export const ValueContainerImage = (props) => {
             </>
         </components.ValueContainer>
     )
-}
\ No newline at end of file
+}
+
+export const validateMomentDate = (date: string, format: string): string => {
+    if(!date || date.length === 0 || date === ZERO_TIME_STRING) return '--'
```

Comment:
We can ommit the `date.length === 0` check

Reply:
Got it and fixed